packages/cli/README.md

```diff
@@ -1932,9 +1932,7 @@ const listExample = (z, bundle) => {
   return z
     .request('https://example.com/api/v2/recipes.json', customHttpOptions)
     .then(response => {
-      if (response.status >= 300) {
-        throw new Error(`Unexpected status code ${response.status}`);
-      }
```

Comment:
We should encourage the use of `throwForStatus` as much as we can (at least until we default its middleware)

Reply:
🤦 I already updated the snippets this was generated from, so I'll revert this and regenerate.

Reply:
Oh, that actually results in the same diff - which makes sense as on pre-commit like you said `README.md` is updated with the updated snippets.